### PR TITLE
small code cleanup

### PR DESCRIPTION
**Pull request by psyborg55**
Signed-off-by: Tomislav Požega <pozega.tomislav@gmail.com>

### CODE REVIEW

**Comment by erikarn** [comment]
Hi,

So 11b rates to legay mode should be configurable - a lot of times you end
up getting a range boost from using CCK rates that you just can't get at 6m
ofdm.


-a

**Comment by olerem** [comment]
grr... i wonted to ack only some of patches

File: target_firmware/wlan/ah.h [hunk 1]
```diff
@@ -136,9 +136,6 @@ typedef enum {
 #endif
 } HAL_RX_FILTER;
 
-#define CHANNEL_QUARTER 0x8000  /* Quarter rate channel */
-#define CHANNEL_HALF    0x4000  /* Half rate channel */
-
```
Comment (olerem) [inline target_firmware/wlan/ah.h]: this one is ok

File: target_firmware/wlan/include/wlan_hdr.h [hunk 2]
```diff
@@ -96,8 +96,8 @@ struct ieee80211com_target {
 	a_uint8_t     pad;
 };
 
-#define ATH_NODE_MAX 8       /* max no. of nodes */
-#define ATH_VAP_MAX  2       /* max no. of vaps */
+#define ATH_NODE_MAX 16       /* max no. of nodes */
+#define ATH_VAP_MAX  8       /* max no. of vaps */
```
Comment (olerem) [inline target_firmware/wlan/include/wlan_hdr.h]: The tests description and the change are not equal. If only 3 VAP was tested, why are you set it to 8?
IMO, single station connected to 3 VAP is not enough.

File: target_firmware/wlan/include/wlan_hdr.h [hunk 3]
```diff
@@ -96,8 +96,8 @@ struct ieee80211com_target {
 	a_uint8_t     pad;
 };
 
-#define ATH_NODE_MAX 8       /* max no. of nodes */
```
Comment (olerem) [inline target_firmware/wlan/include/wlan_hdr.h]: ATH_NODE_MAX 16 is not described in the test.

File: target_firmware/wlan/ar5416Phy.c [hunk 4]
```diff
@@ -70,8 +70,8 @@ RATE_TABLE_11N ar5416_11ngRateTable = {
      /*    2 Mb [1]  */ {  TRUE_ALL_11N, TRUE_ALL_11N, WLAN_PHY_CCK,     2000,  1900,  0x1a, 0x04,    4,    1,     1,      1,       1,    1,     1,     1,  3, 7,    0, FALSE},
      /*  5.5 Mb [2]  */ {  TRUE_ALL_11N, TRUE_ALL_11N, WLAN_PHY_CCK,     5500,  4900,  0x19, 0x04,    11,   2,     2,      2,       2,    2,     2,     2,  3, 7,    0, FALSE},
      /*   11 Mb [3]  */ {  TRUE_ALL_11N, TRUE_ALL_11N, WLAN_PHY_CCK,     11000, 8100,  0x18, 0x04,    22,   3,     3,      2,       3,    3,     3,     3,  3, 7,    0, TRUE},
-     /*    6 Mb [4]  */ {  FALSE,        FALSE,        WLAN_PHY_OFDM,    6000,  5400,  0x0b, 0x00,    12,   4,     2,      1,       4,    4,     4,     4,  3, 7,    0, FALSE},
-     /*    9 Mb [5]  */ {  FALSE,        FALSE,        WLAN_PHY_OFDM,    9000,  7800,  0x0f, 0x00,    18,   4,     3,      1,       5,    5,     5,     5,  3, 7,    0, FALSE},
```
Comment (olerem) [inline target_firmware/wlan/ar5416Phy.c]: Did you tested it in single rate setup? Was this rates ever used in the test?

File: target_firmware/wlan/include/wlan_hdr.h (hunk 3 above)
Comment (psyborg55) [reply target_firmware/wlan/include/wlan_hdr.h]: i suspected this relates to no. of stations allowed to connect, and i still haven't verified if it imposes the mentioned limit.

File: target_firmware/wlan/include/wlan_hdr.h (hunk 2 above)
Comment (psyborg55) [reply target_firmware/wlan/include/wlan_hdr.h]: that was at the time pr was opened. later i managed to run 8 VAPs with 5 clients connected to these...

File: target_firmware/wlan/ar5416Phy.c (hunk 4 above)
Comment (psyborg55) [reply target_firmware/wlan/ar5416Phy.c]: lowered output power on both endpoints so the devices sync at 6Mbps. i could navigate through luci pages without problems (test done on openwrt device)

**Comment by psyborg55** [comment]
this PR was opened accidentally, i was trying to merge it into my tree but clicked too fast.

**Pull request closed without merging**